lib/registries/addon/drafts/draft/-components/tags-manager/component.ts

```diff
@@ -0,0 +1,40 @@
+import { tagName } from '@ember-decorators/component';
+import Component from '@ember/component';
+import { action } from '@ember/object';
+import { ChangesetDef } from 'ember-changeset/types';
+import config from 'ember-get-config';
+import { layout } from 'ember-osf-web/decorators/component';
+import pathJoin from 'ember-osf-web/utils/path-join';
+import template from './template';
+
+const {
+    OSF: { url: baseUrl },
+} = config;
+
```

Comment:
it'd be nice to define and export a `MetadataTagsManager` interface. Then import that in the `RegistriesTagsWidget` that way it's clear that `RegistriesTagsWidget` also work with the metadata manager. Having the manager in `RegistriesTagsWidget` typed correctly can help prevent errors in the future.